news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+Even though it's the first of it's kind in the ranked section, Transhumanist sets an extremely high bar and proves that these kinds of maps are not only rankable, but exceptional in terms of mapping quality and providing an enjoyable player experience. This map is arguably one of Kamuy's best maps so far and we're proud to showcase it in spotlights!
+
+*written by [-MysticEyes](https://osu.ppy.sh/users/6253266)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/experiment.jpg)](https://osu.ppy.sh/beatmapsets/914341)
+
+[Curiossity](https://osu.ppy.sh/users/6490035) strikes again with another insanely innovative and creative masterpiece.
+
+Every bit of this map is thought out with very particularly detailed and visually appealing patterning and speed changes, which gives the map a great amount of replay value, especially for a marathon. The map's excitement is maintained throughout the map, making for a roller-coaster of an experience. 
+
+It never gets boring, not even during the slower sections, in which the player must pay very close attention to the long note releases at a slow scroll speed. This map thus deserves its top spot in the spotlights, and we hope to see Curiossity continue to produce spectacular maps like this one in the future.
+
+*written by [-mint-](https://osu.ppy.sh/users/8976576)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/every.jpg)](https://osu.ppy.sh/beatmapsets/905599)
+
+If you had to describe this map with one word, it'd be something along the lines of... Intricate. Detailed. Accurate. Clean. Recently we have seen a rise of mappers in osu!mania with this approach in mind, [Monheim](https://osu.ppy.sh/users/6814203) and [Vortex-](https://osu.ppy.sh/users/4999669) being well-known for their precision and accuracy in their maps. 
+
+This map, in particular, accomplishes things that many mappers most likely would not be able to accomplish if they were to map this - it is musically accurate in that it expresses many instruments and melody lines at once, is delicate and clean to play, and provides a challenge for all sorts of players. These remain consistent through each difficulty of the mapset. This map thus deserves its spot in this season's spotlights, and be on the lookout - we most definitely will see more by these two extraordinarily talented mappers in the future.
+
+*written by [-mint-](https://osu.ppy.sh/users/8976576)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/oceanus.jpg)](https://osu.ppy.sh/beatmapsets/539179)
+
+It has been a while before [Level 51](https://osu.ppy.sh/users/3617847) made his appearance in mapping ground, and this time he brings the community with a great 6k mapset, Oceanus. 
```

Comment:
```suggestion
It has been a while since [Level 51](https://osu.ppy.sh/users/3617847) made his appearance in the mapping scene, and this time he brings the community a great 6k mapset, Oceanus. 
```